sdks/RepoToolset/tools/Sign.proj

```diff
@@ -19,6 +25,8 @@
     </Exec>
 
     <ItemGroup>
+      <SignToolArgs Condition="'$(OrchestrationMode)' == 'true'" Include='-orchestrationmode' />
+      <SignToolArgs Condition="'$(OrchestrationConfigFile)' != ''" Include='-orchestrationconfigfile "$(OrchestrationConfigFile)"' />      
```

Comment:
>Condition="'$(OrchestrationConfigFile)' != ''"  [](start = 20, length = 47)

Nit: Flip the order of Include and Condition attributes to match the other optional parameters.